_data/contributions.yml

```diff
@@ -1,4 +1,7 @@
 devs:
+  - github_url: "https://github.com/imsanjaysoni"
+    name: "Sanjay Soni 👨‍💻"
+    avtar: "https://avatars3.githubusercontent.com/u/20748185?s=400&u=3b4fbd50461e0381978ca48154bb24419a74d750&v=4"
```

Comment:
Hey! there is a spell mistake, it's avatar not avtar.

Reply:
Corrected